### PR TITLE
DRD4TV-3571: Added custom font support for en-JP locale (English in Japan)

### PR DESCRIPTION
**Pull request by Djspaceg**
Enyo-DCO-1.1-Signed-off-by: Blake Stephens <blake.stephens@lge.com>

### CODE REVIEW

File: src/fonts.js [hunk 1]
```diff
@@ -55,37 +55,51 @@ function funLocaleSpecificFonts () {
...
-					'U+3000-U+303F,' +
-					'U+3040-U+309F,' +
-					'U+30A0-U+30FF,' +
-					'U+3200-U+33FF,' +
-					'U+3400-U+4DBF,' +
-					'U+4E00-U+9FFF,' +
-					'U+E000-U+FAFF,' +
-					'U+FF00-U+FFEF'
+					'U+0-FF,' +
+					'U+2E80-2EFF,' +
```
Comment (yeram) [inline src/fonts.js]: Hi @Djspaceg, just curious, is it same behavior with excluding 'U+' from the end range?

Comment (webOS101) [inline src/fonts.js]: https://drafts.csswg.org/css-fonts-3/#descdef-font-face-unicode-range

It looks like it was wrong before.

Comment (Djspaceg) [inline src/fonts.js]: Yes @yeram, it was wrong before.
We fixed the glitch

**Comment by yeram** [comment]
Got it, thanks!

**Pull request merged**